docs/requirements.txt

```diff
@@ -71,7 +75,8 @@ sphinxcontrib-qthelp==1.0.3
     # via sphinx
 sphinxcontrib-serializinghtml==1.1.5
     # via sphinx
-urllib3==2.0.4
-    # via requests
-zipp==3.16.2
-    # via importlib-metadata
+urllib3==1.26.16
```

Comment:
Do we know why the downgrade occurred here? Is it because this was generated with Python 3.11?

Reply:
Not sure, it would be weird for that to result in a downgrade though. I can re-run `pip-compile` and see if it changes

Reply:
Confirmed they build fine.